core/math/convex_hull.cpp

```diff
@@ -62,8 +62,9 @@ subject to the following restrictions:
 #include "core/math/aabb.h"
 #include "core/math/math_defs.h"
 #include "core/os/memory.h"
+#include "core/string/print_string.h"
+#include "core/templates/oa_hash_map.h"
 #include "core/templates/paged_allocator.h"
-
 #include <string.h>
```

Comment:
Keep that separation line, it's required by our style guide: https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Godot includes and system includes should stay separate.